setcontext/main.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+import sys
+import pathlib
+
+#Add pytest
```

Comment:
nit: good practice to use the keyword TODO as most IDEs understand that.
`#TODO add pytest`

https://google.github.io/styleguide/pyguide.html#312-todo-comments

Reply:
Done